setup.py

```diff
@@ -45,7 +45,7 @@
     include_package_data=True,
     install_requires=[
         "pytket ~= 1.23",
-        "qsharp ~= 0.28.291394",
+        "qsharp ~= 1.0.33",
```

Comment:
```suggestion
        "qsharp ~= 1.0",
```
Further 1.x releases shouldn't break compatibility.